sdk/log.py

```diff
@@ -48,8 +49,39 @@ def print_data(self, data):
             else:
                 print("\t", key, ": ", value)
 
+        if data["status"] == "COMPLETED":
+            self.plot_graphs(data["jobParameters"], batchSteps)
+
     def time_difference(self, start_time, end_time):
         start_date_time_obj = datetime.fromisoformat(start_time.split("+")[0])
         end_date_time_obj = datetime.fromisoformat(end_time.split("+")[0])
         tdelta = end_date_time_obj - start_date_time_obj
         return tdelta.total_seconds()
+
+    def plot_graphs(self, data, batchSteps):
+        plot_data = {}
+        for id, batch_step in batchSteps.items():
+            step_name = batch_step["step_name"]
+            if step_name not in plot_data:
+                timeTaken = batch_step["timeTaken"]
+            if data[step_name]:
+                size = self.process_data(data[step_name])
+            try:
+                plot_data[step_name] = ((int(size) / 1000000) * 8) / timeTaken
+            except(Exception):
```

Comment:
I would add maybe some std print saying you were unable to render graph for a specific step name

Reply:
Added a check to display a message for skipped steps.